README.NewResourceFromTerraform.md

```diff
@@ -615,6 +615,11 @@ update to date, we need to add the new CRDs and samples to existing doc.
     `https://cloud.google.com` (i.e.
     `https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances`).
     Add an additional paragraph/table if needed.
+1.  Update the URL in "Before you begin" section, points to the "Prerequisites" or 
+    "Before you begin" section in the service guide. Make sure that the URL
+    is working by adding the partial URL to
+    `https://cloud.google.com` (i.e.
+    `https://cloud.google.com/spanner/docs/create-query-database-console#before`).
```

Comment:
IIUC, this sentence implies that the resource contributors should:
1. Use partial URL in this section.
2. Test out whether the partial URL works by prepending `https://cloud.google.com`?

I like the idea of verification, and I understand that it is copied from the existing doc above, but I think it'll be more clear if:

1. We clarify what is the "URL" and what is the "partial URL". You seem to refer to the same thing here? I'll probably just call it "partial URL" from the beginning. E.g.

Replace the partial URL "/anthos/clusters/docs/multi-cloud/attached/eks/how-to/attach-cluster" with the partial URL pointing to the "Prerequisites" or "Before you begin" section in the resource guide. Make sure it works when prepending `https://cloud.google.com` to the partial URL. For example, if the partial URL is updated to `/spanner/docs/create-query-database-console#before`, verify if `https://cloud.google.com/spanner/docs/create-query-database-console#before` is a working URL.

2.  We suggest the verification as the last step of this section as there are multiple steps using the concept of "partial URL", and I think it'll be easier if we use a single step to talk about the verification.

Reply:
I updated the existing sentences regarding partial URL, thanks for the suggestion and example! I do not think it's necessary to duplicate that in this newly added instruction.